Library/ViewModels/PledgeViewModel.swift

```diff
@@ -3,13 +3,23 @@ import KsApi
 import Prelude
 import ReactiveSwift
 
+public typealias PledgeViewData = (
+  project: Project,
+  reward: Reward,
+  isLoggedIn: Bool,
+  total: Double
+)
+
 public protocol PledgeViewModelInputs {
   func configureWith(project: Project, reward: Reward)
+  func pledgeAmountDidUpdate(to amount: Double)
+  func shippingAmountDidUpdate(to amount: Double)
   func viewDidLoad()
 }
 
 public protocol PledgeViewModelOutputs {
-  var reloadWithData: Signal<(Project, Reward, Bool), Never> { get }
+  var reloadWithData: Signal<PledgeViewData, Never> { get }
+  var updateWithData: Signal<PledgeViewData, Never> { get }
```

Comment:
`reload` and `update` are fairly similar in naming... suggestions?

Reply:
Looks like `updateWithData` is basically just 1. reloading the data source and 2. re-configuring the summary cell. Is it possible to split up those two things and have `updateWithData` just do 2.? Then we can rename it to `configureSummaryCell` or something similar.

Reply:
Well, currently, `reloadWithData` also _reloads_ the `UITableView` so we can't update the data source with that because we only want to silently update it and configure the cell. I also considered `pledgeViewDataAndReload` which would be like `(PledgeViewData, Bool)` where the `Bool` is whether or not to reload the table view. If we did this then we could have just one other signal to reconfigure the cell after the table view is reloaded. However - I am weary of running into that race condition we had before in another PR 🤔

Reply:
Ah okay I see! Yeah this makes sense then.